packages/spor-line-tag-react-native/src/LineIcon.tsx

```diff
@@ -0,0 +1,214 @@
+    variant === "long-distance-train" ||
+    variant === "airport-express-train"
+  );
+}
+
+const getIcon = (
+  variant: LineIconVariantProps,
+  size: LineIconSizeProps,
+  iconColor: string
+) => {
+  if (isTrain(variant)) {
+    switch (size) {
+      case "sm":
+        return <TrainFill18Icon color={iconColor} />;
+      case "md":
+        return <TrainFill24Icon color={iconColor} />;
+      case "lg":
+        return <BusFill30Icon color={iconColor} />;
+    }
+  } else {
+    switch (variant) {
+      case "vy-bus":
+        switch (size) {
+          case "sm":
+            return <ExpressBusFill18Icon color={iconColor} />;
+          case "md":
+            return <ExpressBusFill24Icon color={iconColor} />;
+          case "lg":
+            return <ExpressBusFill30Icon color={iconColor} />;
+        }
+      case "local-bus":
+        switch (size) {
+          case "sm":
+            return <BusFill18Icon color={iconColor} />;
+          case "md":
+            return <BusFill24Icon color={iconColor} />;
+          case "lg":
+            return <BusFill30Icon color={iconColor} />;
+        }
+      case "ferry":
+        switch (size) {
+          case "sm":
+            return <FerryFill18Icon color={iconColor} />;
+          case "md":
+            return <FerryFill24Icon color={iconColor} />;
+          case "lg":
+            return <FerryFill30Icon color={iconColor} />;
+        }
+      case "subway":
+        switch (size) {
+          case "sm":
+            return <SubwayFill18Icon color={iconColor} />;
+          case "md":
+            return <SubwayFill24Icon color={iconColor} />;
+          case "lg":
+            return <SubwayFill30Icon color={iconColor} />;
+        }
+      case "tram":
+        switch (size) {
+          case "sm":
+            return <TramFill18Icon color={iconColor} />;
+          case "md":
+            return <TramFill24Icon color={iconColor} />;
+          case "lg":
+            return <TramFill30Icon color={iconColor} />;
+        }
+      case "alt-transport":
+        switch (size) {
+          case "sm":
+            return <AltTransportFill18Icon color={iconColor} />;
+          case "md":
+            return <AltTransportFill24Icon color={iconColor} />;
+          case "lg":
+            return <AltTransportFill30Icon color={iconColor} />;
+        }
+      case "walk":
+        switch (size) {
+          case "sm":
+            return <WalkFill18Icon color={iconColor} />;
+          case "md":
+            return <WalkFill24Icon color={iconColor} />;
+          case "lg":
+            return <WalkFill30Icon color={iconColor} />;
+        }
+      default:
+        return null;
+    }
+  }
+};
+
+export const LineIcon = ({
+  variant,
+  size,
+  travelOrInfo,
+  iconColor = "white",
+  ...props
+}: LineIconProps) => {
+  const { style } = useRestyle(restyleFunctions, {
+    variant,
+    ...props,
+  });
+  const iconType = size + "-" + travelOrInfo;
```

Comment:
En småting: Bruk template strings for å konkatenere strenger på denne måten – jeg synes det leser bedre.

```suggestion
  const iconType = `${size}-${travelOrInfo}`;
```